src/lib/_boxui.scss

```diff
@@ -376,11 +376,15 @@
             opacity: 0.6;
             outline: none;
 
-            &.close-btn.default-close-btn {
-                width: 24px;
-
-                path.icon {
-                    fill: $white;
+            &.default-close-btn {
+                svg {
+                    height: 24px;
+                    pointer-events: none;
```

Comment:
I'm assuming the event is handled by the parent?

Reply:
Yeah